jenkins/deploy-clio-server.sh

```diff
@@ -141,9 +148,9 @@ stopcontainer
 # move /app.new to /app
 # create pos log directory if it doesn't already exist
 ${SSHCMD} <<-EOF
-  ([ ! -d ${APP_DIR}.old ] || sudo mv ${APP_DIR}.old ${APP_DIR}.old.old) &&
-  ([ ! -d ${APP_DIR} ] || sudo mv ${APP_DIR} ${APP_DIR}.old) &&
-  sudo mv ${APP_DIR}.new ${APP_DIR} &&
+  ([ ! -d ${PREVIOUS_APP_DIR} ] || sudo mv ${PREVIOUS_APP_DIR} ${PREVIOUS_APP_BACKUP_DIR}) &&
```

Comment:
Short and honest answer is that this is how the deploy script for our Cromwell did things. @hjfbynara, can we configure the Jenkins user on our GCE instances to not need `sudo` for modifying files in the root directory?